tests/queries/0_stateless/02801_backup_native_copy.reference

```diff
@@ -1,4 +1,8 @@
 BACKUP TABLE data TO S3(s3_conn, \'backups/default/data_native_copy\') SETTINGS allow_s3_native_copy=true	1
 BACKUP TABLE data TO S3(s3_conn, \'backups/default/data_no_native_copy\') SETTINGS allow_s3_native_copy=false	0
+BACKUP TABLE data TO S3(s3_conn, \'backups/default/data_native_copy_inc\') SETTINGS allow_s3_native_copy=true, base_backup=S3(s3_conn, \'backups/default/data_native_copy\')	0
```

Comment:
Shouldn't it be `1` here (instead of `0`)?

Reply:
No, because it has base backup, and nothing will be copied at all.

Reply:
Please add a short description to the test about why we expect this or that number of `S3CopyObject`.

Reply:
Done